cli/internal/run/run_state.go

```diff
@@ -228,7 +228,7 @@ func (r *RunState) Close(Ui cli.Ui, filename string) error {
 		name = filename
 	}
 	if outputPath != "" {
-		if err := fs.CopyFile(outputPath, name, fs.DirPermissions); err != nil {
+		if err := fs.CopyFile(fs.LstatCachedFile{Path: fs.AbsolutePath(outputPath)}, name); err != nil {
```

Comment:
We typically don't cast, we `fs.UnsafeToAbsolutePath()`

Reply:
👍🏼  Good callout